app/helper/http/get.go

```diff
@@ -0,0 +1,28 @@
+package http
+
+import (
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+)
+
+func Get(client *http.Client, url string, headers map[string]string, log *log.Entry) (responseBodyAsBytes []byte, err error) {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		log.Errorf("Error while creating http request struct. Error: [%v]", err)
+		return responseBodyAsBytes, err
+	}
+
+	for key, value := range headers {
+		req.Header.Set(key, value)
+	}
+
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Errorf("Error while sending request to server. Error: [%v]", err)
+		return responseBodyAsBytes, err
+	}
+
+	responseBodyAsBytes, _ = ioutil.ReadAll(resp.Body)
```

Comment:
Unclosed stream;
Unchecked error;
io/ioutil is deprecated;
If possible, avoid using ReadAll;

Maybe add an `interface{}` argument to for a struct ref to which you can
`json.NewDecoder(resp.Body).Decode(struct)` the response. 

If helpers for deserialization is an established pattern you can pass the deserialization function as well.